File: problemSet/template.py
"""
Codeforces Round #334 (Div. 2)

Problem 37 A. Towers

@author yamaton
@date 2015-12-01
"""

import itertools as it
import functools
import operator
import collections
import math
import sys


def solve(xs, n):
    ...


# def pp(*args, **kwargs):
#     return print(*args, file=sys.stderr, **kwargs)


def main():
    n = int(input())
    a, b, c = map(int, input().split())
    xs = [int(c) for c in input().split()]
    pairs = [tuple(map(int, input().split())) for _ in range(n)]
    assert len(xs) == n
    result = solve(xs, n)
    print(result)


if __name__ == '__main__':
    main()

File: problemSet/template.scala
/**
  * Codeforces Round #332 (Div. 2)
  *
  * Problem 599 A.
  *
  * author:  yamaton
  * date:    2015-11-20
  *
  */

object CF599A {
  def solve(xs: List[Int], n: Int): Int = {
    
  }


  def main(args: Array[String]) {
    val n = io.StdIn.readInt()
    val Array(a, b) = io.StdIn.readLine.split(" ").map(_.toInt)
    val xs = io.StdIn.readLine.split(" ").map(_.toInt).toList
    assert(xs.length == n)
    val ans = solve(xs, n, a, b)
    println(ans)
  }
}
